content/en/network_performance_monitoring/guide/manage_traffic_costs_with_npm.md

```diff
@@ -0,0 +1,64 @@
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/datacenter.png" alt="Group flows by datacenter">}}
+    An increase in your traffic bill is almost always tied to an increase in one of these types of traffic. In many cases you may want to group traffic by asymmetric search terms - meaning, you want to see the source of the traffic in terms of one tag, but want to see the destination in terms of another. This kind of asymmetric query can be used to identify costly dependencies between your on-prem datacenters and cloud regions, 
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/aws_account.png" alt="Identify dependencies between datacenters and cloud regions">}}
+    as well as between clouds. One particularly helpful view is to group the source of the traffic by service, and the destination across availability zones. This will help you identify particularly noisy and expensive services that are communicating across multiple availability zones, like what we found during our migration.
+
+2. From here, isolate services that have the most traffic volume across multiple AZs. You can use filters inside the search bars to narrow down your query - so, for example, you can show only services that originate inside of one availability zone, and are sending traffic to any availability zone other than that one.
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/service_availability_zone.png" alt="Highlight any services that communicated across AZs">}}
+    The query above highlights any services that communicate across AZs, from us-east4-a to anywhere else. The chattiest services surfaced by this query may contribute heavily to overall cross-AZ traffic volume, and we can narrow it down even further by searching for one particular service, grouping the traffic by availability zone.
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/single_service.png" alt="Search for a single service.">}}
+
+3. Similarly, you can use the team tag to identify engineering teams that generate the most, for instance, cross-regional traffic 
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/team_region.png" alt="Use the team tag.">}}
+or monitor your own team’s output specifically.
+{{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/region_region.png" alt="Use the region tag.">}}
+
+4. To monitor costs from external traffic, scope your destination endpoints to public IPs using the IP Type facet.
+    {{< img src="network_performance_monitoring/guide/manage_traffic_costs_with_npm/scope_destination_points.png" alt="Use the type facet.">}}
+    Then group your destination by `dns` to break down external traffic volume by where it’s going. Although you cannot install an agent on public servers, we are able to resolve external endpoints to human-readable domain names using DNS resolution. 
```

Comment:
```suggestion
    Then group your destination by `domain` to break down external traffic volume by where it’s going. Although you cannot install an agent on public servers, we are able to resolve IPs representing external and cloud endpoints to human-readable domain names. 
```

Reply:
we can link [external and cloud endpoints to human-readable domain names][https://www.datadoghq.com/blog/dns-resolution-datadog/]